aptos-move/framework/aptos-framework/sources/delegation_pool.move

```diff
@@ -346,6 +350,8 @@ module aptos_framework::delegation_pool {
         allowlist: SmartTable<address, bool>,
     }
 
+    struct DelegationPermission has copy, drop, store {}
```

Comment:
If I understand correctly , there are (at least) two different roles here:

DelegationPoolManagementPermission (i.e. for delegation pool owners)
and
StakingToDelegationPoolPermission (i.e. for folks staking their fudns to delegation pool nodes. Also maybe this should be the same permission as regular staking, i.e. differentiating between StakingToDelegationPoolPermission and StakingPermission might be unnecessary